package.json

```diff
@@ -37,7 +37,7 @@
   },
   "devDependencies": {
     "@types/express": "^4.11.1",
-    "@types/node": "^10.0.9",
+    "@types/node": "^14.14.41",
```

Comment:
14.14.41 is the newest version, but I can't imagine there being any significant-to-this-project changes between 14.14 and 14.16 :smile: